lib/simple_saml/simple_saml.ex

```diff
@@ -58,15 +89,40 @@ defmodule SimpleSaml do
+      iex> {:ok, {root, assertion}} = SimpleSaml.parse_response(saml_response)
+      iex> SimpleSaml.verify_and_validate_response(root, assertion, public_key, skip_time_conditions?: true)
+      :ok
+
   """
   @spec verify_and_validate_response(SimpleXml.xml_node(), Assertion.t(), public_key()) ::
           :ok | {:error, any()}
-  def verify_and_validate_response(root_node, %Assertion{} = assertion, public_key) do
-    with :ok <- SimpleXml.verify(root_node, public_key),
-         :ok <- Assertion.validate_time_conditions(assertion) do
+  def verify_and_validate_response(root_node, %Assertion{} = assertion, public_key, opts \\ []) do
+    skip_time_conditions? = Keyword.get(opts, :skip_time_conditions?, false)
+
+    with :ok <- verify_xml(root_node, public_key),
+         :ok <- validate_time_conditions(assertion, skip_time_conditions?) do
       :ok
     else
       {:error, reason} -> {:error, reason}
     end
   end
+
+  defp validate_time_conditions(_assertion, true), do: :ok
+  defp validate_time_conditions(assertion, _), do: Assertion.validate_time_conditions(assertion)
+
+  defp verify_xml(root_node, public_key) do
+    with {:error, _} <- SimpleXml.verify(root_node, public_key),
```

Comment:
@zoldar: Sorry, I missed this earlier.  It feels odd to expect an error in the normal flow.  I'm going to have to dig through this a bit more later today.  Since I already merged this PR, I'll just make follow-up changes, if necessary.

Reply:
Sure, understood. Thanks!